pyseir/rt/infer_rt.py

```diff
@@ -16,37 +16,46 @@
 rt_log = structlog.get_logger(__name__)
 
 
-def run_rt_for_fips(fips, figure_collector=None):
+def run_rt_for_fips(
+    fips, include_deaths=False, include_testing_correction=True, figure_collector=None
+):
     """Entry Point for Infer Rt"""
 
     # Generate the Data Packet to Pass to RtInferenceEngine
     input_df = _generate_input_data(
-        fips, include_testing_correction=True, figure_collector=figure_collector
-    )
-
-    # Save a reference to instantiated engine (eventually I want to pull out the figure generation
-    # and saving so that I don't have to pass a display_name and fips into the class
-    engine = RtInferenceEngine(
-        data=input_df, display_name=_get_display_name(fips), fips=fips, include_deaths=False
+        fips=fips,
+        include_testing_correction=include_testing_correction,
+        include_deaths=include_deaths,
+        figure_collector=figure_collector,
     )
+    if input_df.dropna().empty:
+        rt_log.warning(event="Infer Rt Skipped. No Data Passed Filter Requirements:", fips=fips)
```

Comment:
I think it's clearer here if you return at this point, then everything below doesn't ahve to be indented

Reply:
Done. I have a weak preference to use if/else logic with the return at the end when there is only one logic branching event, but it's clear we could add additional short-circuit events in here around how output_df returns.